<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeFilter</class>
 <widget class="QDialog" name="TimeFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>234</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>TimeFilter</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>301</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="title">
    <string>选择时间段</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>40</number>
    </property>
    <item>
     <widget class="QRadioButton" name="NearlyADay">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>近一天</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="NearlyAWeek">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>近一周</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="NearlyAMonth">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>近一月</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="NearlyAYear">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>近一年</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="All">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>所有</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
